src/clarity-angular/emphasis/alert/alerts.spec.ts

```diff
@@ -139,6 +138,17 @@ describe("Alerts component", function() {
             fixture.detectChanges();
             expect(compiled.querySelectorAll("clr-alerts-pager").length).toBe(0);
         });
+
+        it("makes the previous alert active when the current alert is removed from the DOM", function() {
+            fixture.componentInstance.currentAlertIndex = 1;
+            fixture.detectChanges();
+            expect(alertElements[0].classList).toContain("alert-hidden");
+
+            fixture.componentInstance.alertInstances.reset([this.alert]);
+            fixture.detectChanges();
+
+            console.log(fixture.componentInstance.currentAlertIndex);
```

Comment:
Make this an `expect` to confirm the result.

Reply:
good catch. this means that my test didn't run itself. let me check and confirm if this change actually works.